src/pages/ChatPage/components/ChatHeader.tsx

```
@@ -0,0 +1,193 @@
+                <p className="text-body font-medium text-neutral-foreground">
+                  Ahmad Sami
+                </p>
+                <p className="text-caption text-success">{t("chat.online")}</p>
+              </div>
+            </div>
+
+            <div className="flex items-center gap-3 text-neutral" ref={menuRef}>
+              <button
+                className="hover:text-primary"
+                onClick={() => setShowCallDialog(true)}
+              >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)